languages/python/default.nix

```diff
@@ -49,6 +49,79 @@ in {
       defaultText = lib.literalExpression "pkgs: [ ]";
       example = lib.literalExpression "pkgs: [ pkgs.requests ]";
     };
+    config = lib.mkOption {
+      type = with lib.types; nullOr lines;
+      description = ''
+        Python interpreter startup configuration. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = null;
+      defaultText = lib.literalExpression "null";
+      example = lib.literalExpression ''
+        import numpy as np
+      '';
+    };
+    configPath = lib.mkOption {
+      type = lib.types.path;
+      description = ''
+        Python interpreter startup configuration file path. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = "${config.xdg.configHome}/python/startup.py";
+      defaultText = "$XDG_CONFIG_HOME/python/startup.py";
+      example =
+        lib.literalExpression
+        ''"${config.xdg.configHome}/pythonrc.py"'';
+    };
+    historyPath = lib.mkOption {
+      type = with lib.types; nullOr path;
+      description = ''
+        Python interpreter history file path. See
+        <https://docs.python.org/3.13/using/cmdline.html#envvar-PYTHON_HISTORY>
+        for details.
+        This option is only available in Python 3.13.
+      '';
+      default = null;
+      defaultText = lib.literalExpression "null";
+      example = lib.literalExpression ''
+        "${config.xdg.stateHome}/python_history"
+      '';
+    };
+    enableColors =
+      lib.mkEnableOption ''
+        colors in the interpreter. See
```

Comment:
It seems the text here is not complete.

Reply:
I’m not sure if this should even be a `mkEnableOption`. You _can_ leave the variable undefined which I presume makes it detect a terminal to enable color (the docs page is unclear on this—maybe try it out once there’s a build with colors).

Reply:
The text should probably include a mention of the alternative global variables `NO_COLOR` and `FORCE_COLOR`, which the user can define themselves.